netty/src/main/java/io/grpc/netty/InternalProtocolNegotiators.java

```diff
@@ -69,4 +71,12 @@ public void close() {
     
     return new TlsNegotiator();
   }
+
+  public static ChannelHandler waitUntilActiveHandler(ChannelHandler next) {
```

Comment:
some javadoc would be helpful

Reply:
Added a dummy comment, but this is an accessor and not meant for normal usage.